src/Symfony/Bridge/Twig/Extension/SerializeExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Jesse Rushlow <jr@rushlow.dev>
+ */
+class SerializeExtension extends AbstractExtension
```

Comment:
I would call it `SerializerExtension` to be consistent with the component name

Reply:
done